apache/ports.conf.in

```diff
@@ -0,0 +1,9 @@
+Listen ${APACHE_PORT}
+
+<IfModule mod_ssl.c>
```

Comment:
mmh, I would not use an SSL termination from within the application container (unless you really want to do end-to-end SSL also within the container environment), but put a reverse proxy with the SSL termination in front of it (ALB, etc). And I guess that you are not copying any SSL certificate in the container, or mounting it from any location, so this will probably get some errors at some point.